ccarlosm/coditra-discography
# Task-1-Generate application basics

Problem
* Set up deployment
* Test the status of App

Solution
* Install artisan api for Sanctum
* Set up Laravel Sail
* We create a simple test to check the application status
